docs/index.md

```diff
@@ -92,13 +92,13 @@ installation.  If you are on a 32-bit installation, please substitute "i386" for
 First, install the Docker client binary:
 
 ```
-curl https://get.docker.com/builds/Windows/x86_64/docker-latest > /bin/docker
+$ curl -L https://get.docker.com/builds/Windows/x86_64/docker-latest > /bin/docker
```

Comment:
This URL is returning an image of a sinking ship that says 404 content not found on top of it.